testcases/kernel/security/selinux/selinux.c

```diff
@@ -0,0 +1,55 @@
+#include <errno.h>
+#include <fcntl.h>
+
+#include "test.h"
+#include <selinux/selinux.h>
+
+
+char *TCID = "selinux";
+int TST_TOTAL = 1;
+
+static void cleanup(void);
+static void setup(void);
+
+static int old_enforcing;
+
+static void setup(void)
+{
+	tst_require_root();
+	tst_tmpdir();
+
+	if (!is_selinux_enabled())
+		tst_brkm(TBROK, cleanup, "selinux not enabled\n");
+
+	old_enforcing = security_getenforce();
+
+	int setenforce_failure = security_setenforce(1);
+
+	if (setenforce_failure)
+		tst_brkm(TBROK, cleanup,
+			"unable to enforce selinux: %d", errno);
+
```

Comment:
If the test is testing that security_setenforce() could be called it shouldn't be done in the setup() and the test should do ```tst_resm(TPASS, ...)``` and ```tst_resm(TFAIL, ...)``` accordingly.

Also setenforce_failure is a poor name for a variable it's overly long and confusing, it should be simply named ```ret``` or ```res```.